Lib/fontTools/colorLib/builder.py

```diff
@@ -328,116 +393,6 @@ def _split_color_glyphs_by_version(
     return colorGlyphsV0, colorGlyphsV1
 
 
-def _to_variable_value(
-    value: _ScalarInput,
-    cls: Type[VariableValue] = VariableFloat,
-    minValue: Optional[_Number] = None,
-    maxValue: Optional[_Number] = None,
-) -> VariableValue:
-    if not isinstance(value, cls):
-        try:
-            it = iter(value)
-        except TypeError:  # not iterable
-            value = cls(value)
-        else:
-            value = cls._make(it)
-    if minValue is not None and value.value < minValue:
-        raise OverflowError(f"{cls.__name__}: {value.value} < {minValue}")
```

Comment:
we are no longer validating the min and max values at build time. These overflow errors will only be caught (as less user-friendly struct.error) later on upon compiling (serializing) the otTables. Ideally we catch these earlier at construction time. Maybe we can add minValue and maxValue class attributes to the _NamedTupleConverters and a converter.validate(value) method that raises OverflowError?

Reply:
I was avoiding modifying otTables; if we're open to that then +1!

Reply:
I'm not spotting this type of thing for other classes; maybe this *should* fail at writeValue time?

Reply:
it does fail at writeValue (compile) time already, with struct.error. maybe that's enough.